### PR TITLE
Feat/lld/live 7082: Add staking_completed track to tezos and polkadot

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

Add staking_completed track to tezos and polkadot

### ❓ Context

- **Impacted projects**:  LLD
- **Linked resource(s)**:  https://ledgerhq.atlassian.net/browse/LIVE-7082?focusedCommentId=333784&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-333784

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx [hunk 1]
```diff
@@ -23,7 +24,35 @@ const Container = styled(Box).attrs(() => ({
...
+    return allValidators
+      .filter(val => transaction.validators.includes(val.address))
+      .map(val => val.identity || val.address);
+  }, [allValidators, transaction.validators]);
+
+  useEffect(() => {
+    if (optimisticOperation && validators) {
+      track("staking_completed", {
+        currency: "DOT",
+        validator: validators,
```
Comment (kallen-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx:48]: [Question] Do we not want 1 validator to be returned here and not an array of validators?

File: apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx [hunk 2]
```diff
@@ -23,7 +24,35 @@ const Container = styled(Box).attrs(() => ({
...
+  source,
+}: StepProps) {
+  const preloaded = usePolkadotPreloadData();
+  const { validators: allValidators } = preloaded;
+
+  const validators = useMemo(() => {
+    return allValidators
+      .filter(val => transaction.validators.includes(val.address))
+      .map(val => val.identity || val.address);
+  }, [allValidators, transaction.validators]);
```
Comment (kallen-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx:42]: [Question] Do we not want to return 1 validator or undefined and not a string[]?

File: apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx (hunk 1 above)
Comment (sarneijim) [reply apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx:48]: I guess not, DOT support a list of validators. Do not ask me why...

Comment (kallen-ledger) [reply apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx:48]: Ah ok, good to know for the future. Thanks.

**Review by kallen-ledger — APPROVED** [review]

**Pull request merged**